cpi-api-v1.html.md.erb

```diff
@@ -94,11 +94,12 @@ Make sure to properly delete created resources if VM cannot be successfully crea
 - **cloud_properties** [Hash]: Cloud properties hash specified in the deployment manifest under VM's resource pool.
 - **networks** [Hash]: Networks hash that specifies which VM networks must be configured.
 - **disk_cids** [Array of strings] Array of disk cloud IDs for each disk that created VM will most _likely_ be attached; they could be used to optimize VM placement so that disks are located nearby.
-- **environment** [Hash]: Resource pool's env hash specified in deployment manifest.
+- **environment** [Hash]: Resource pool's env hash specified in deployment manifest including
+initial properties added by the BOSH director as shown below.
```

Comment:
[Apparently](https://github.com/cloudfoundry/docs-bosh/pull/223#issuecomment-253623469) we're not supposed to wrap lines in these docs.